modules/vba_documents/app/models/vba_documents/upload_submission.rb

```diff
@@ -122,7 +128,7 @@ def consumer_name
     end
 
     def appeals_consumer?
-      /appeals_api/.match?(consumer_name)
+      Regexp.new(APPEALS_API_CONSUMER_PATTERN).match?(consumer_name)
```

Comment:
`Regexp.new(APPEALS_API_CONSUMER_PATTERN)` results in `/appeals_api/`, so no change here. (There is also an existing unit test for this method.) 

I ended up going this route so that `APPEALS_API_CONSUMER_PATTERN` could be used both for this method and for the `ILIKE` PG query in the `not_from_appeals_api` scope.